.github/workflows/pr-differences-mutants.yml

```diff
@@ -68,15 +71,15 @@ jobs:
         uses: stacks-network/actions/stacks-core/mutation-testing/pr-differences@main
         with:
           shard: ${{ matrix.shard }}
-          package-dimension: "small"
+          package: "small"
 
-  # Mutation testing - Execute on PR on big packages that have functions modified (normal run, no shards)
-  pr-differences-mutants-big-normal:
-    name: Mutation Testing - Normal, Big
+  # Mutation testing - Execute on PR on stackslib package (normal run, no shards)
+  pr-differences-mutants-stackslib-normal:
+    name: Mutation Testing - Normal, Stackslib
 
     needs: check-big-packages-and-shards
 
-    if: ${{ needs.check-big-packages-and-shards.outputs.run_big_packages == 'true' && needs.check-big-packages-and-shards.outputs.big_packages_with_shards == 'false' }}
+    if: ${{ needs.check-big-packages-and-shards.outputs.run_stackslib == 'true' && needs.check-big-packages-and-shards.outputs.stackslib_with_shards == 'false' }}
```

Comment:
can you change these conditionals so they're easier to read? ex:
https://github.com/stacks-network/stacks-core/blob/master/.github/workflows/ci.yml#L55-L63

this goes for any `if:` in this PR